### ISSUE TITLE
TypeError: allure.label is not a function

### ISSUE THREAD

**Issue by darkallure** [allure-framework/allure-js#486]
**Describe the bug**
Getting an error when parallel mode is on while using Mocha

**To Reproduce**
1. Set parallel mode to true in mocha.rc config or use "--parallel" flag
2. Run the tests
3. Tests fail due to the following error: Uncaught TypeError: hook.error is not a function

**Expected behavior**
The tests should run successfully in parallel mode with Mocha allure reporter

**Screenshots**
![image](https://user-images.githubusercontent.com/20166110/185746624-5240de8f-9b0e-4a8e-bb36-a6fe5f2b43a2.png)
![image](https://user-images.githubusercontent.com/20166110/185746644-d99610dc-6961-4dc6-abb6-306f6bbca6a5.png)

**Additional context**
the following version packages are used:
"allure-mocha": "^2.0.0-beta.19",
"mocha": "^9.2.1"

**Comment by darkallure** [comment]
@srg-kostyrko

**Comment by lamartire** [comment]
Could you please share more code or provide a repo, where would be possible to reproduce the problem?

**Comment by darkallure** [comment]
[Uploading lmru--chords--functional-tests2.zip…]()
Sure, here is a sample project

**Comment by darkallure** [comment]
@lamartire did you manage to check it ou?

**Comment by darkallure** [comment]
[example-js-allure.zip](https://github.com/allure-framework/allure-js/files/9545659/example-js-allure.zip)

**Comment by epszaw** [comment]
Sorry guys, still working on the the solution

**Comment by darkallure** [comment]
Nice, please let me know in case there is an update, thanks

**Comment by epszaw** [comment]
Sorry for the long delay, but at this moment we're a bit paralyzed due parallel mode nuances. I've opened a PR (mochajs/mocha#4922) to mocha which provides required changes to restore back labels functionality, but I haven't yet got any response from them. Let's wait for the merge.

**Comment by BrianSalesLoft** [comment]
any update on this?
I'm having the same issue

**Comment by baev** [comment]
3.0.0-beta.1 introduced support for parallel mode

**Comment by delatrie** [comment]
The issue has been fixed in 3.0.0-beta.1. If anyone still struggles with using Allure API in parallel mode, feel free to open a new issue or post an update here.
